index.js

```diff
@@ -1,38 +1,39 @@
-const walkTree = require('./walktree');
+var walkTree = require('./src/walktree');
```

Comment:
I changed `const`s to `var`s. There is more than a semantic difference, of course, but I did it here to keep as wide compatibility as possible across Node versions. Node starts treating `const`s differently as of 4.x and earlier.